manifests/addons/values-grafana.yaml

```diff
@@ -14,12 +14,12 @@ podLabels:
 
 # Demo only, so we will have no authentication
```

Comment:
Why do we change this? I think before it was just no login required which seems ideal for demo. Better than fake password?

Reply:
need login to manage datasource, so make the username and password more meaningful than `-`